inc/umock_c/umock_c.h

```diff
@@ -135,9 +135,10 @@ int umock_c_set_call_recorder(UMOCKCALLRECORDER_HANDLE umockc_call_recorder);
 #endif
 
 /* Codes_SRS_UMOCK_C_LIB_01_149: [ REGISTER_UMOCK_ALIAS_TYPE registers a new alias type for another type. ]*/
+/* Codes_SRS_UMOCK_C_LIB_02_001: [ If the types do not have the same size the on_error callback shall be called with UMOCK_C_REGISTER_TYPE_FAILED. */
```

Comment:
>Codes_SRS_UMOCK_C_LIB_02_001 [](start = 3, length = 28)

Can we add a test for this in the int tests at least? If not I'll have to do it when I go and enable traceability (like we did for C util :-)).

Reply:
added

---
In reply to: [365646017](https://github.com/Azure/umock-c/pull/138#discussion_r365646017) [](ancestors = 365646017)